include/verilatedos.h

```diff
@@ -614,8 +614,12 @@ static inline double VL_ROUND(double n) {
 //=========================================================================
 // Time and performance
 
+#include <string>
+
 namespace VlOs {
 
+extern std::string getenvStr(const std::string& envvar, const std::string& defaultValue)
+    VL_MT_SAFE;  ///< Get environment variable
```

Comment:
```suggestion
/// Get environment variable
extern std::string getenvStr(const std::string& envvar, const std::string& defaultValue)
    VL_MT_SAFE;

```

(Since doesn't fit on one line.)